recipes/antlr4-python2-runtime/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "antlr4-python2-runtime" %}
```

Comment:
Any idea why these are two different pypi packages? In conda, we may want to combine them and use the dependency infastructure to build the correct one. Then you can simply `conda install python-antlr` to install.

Reply:
Well, to grammar people, they are [as different as go and javascript](https://github.com/antlr/antlr4/tree/master/runtime)! Indeed, we could even build them directly from the upstream repo. I guess since they install the same thing (`antlr`) to `site-packages`, that would not be unreasonable.

Also, humorously, `antlr 4.7.1` just dropped within the last few minutes:
https://github.com/antlr/antlr4/releases/tag/4.7.1

However, the `jar` [hadn't been updated](http://www.antlr.org/download/).

Reply:
It may be best to ask an antlr developer their recommendations on the packaging. I assume they have dealt with this with some other package managers.

Reply:
It appears the antlrX (after antlr2) is the most common nomenclature:
- https://pkgs.org/download/antlr
- https://pkgs.org/download/antlr3
- https://pkgs.org/download/antlr4
- https://packages.debian.org/search?keywords=antlr
- https://admin.fedoraproject.org/pkgdb/package/rpms/antlr
- https://admin.fedoraproject.org/pkgdb/package/rpms/antlr3
- https://admin.fedoraproject.org/pkgdb/package/rpms/antlr4

This is consistent with what is proposed in this PR.

For the runtimes, at least the debian packages appear to favor `python-antlr4` and `python3-antlr4`. It might indeed make sense to combine them into one name.

Reply:
Yeah, I was thinking the same thing.  

Debian doesn't really give a good example because in Debian all packages are split to `python` and `python3`. 

My preference would be to make a single package. Otherwise any package that depends on antlr will have to have 

```
- antlr4-python2-runtime # [py2k]
- antlr4-python3-runtime # [py3k]
```

in the dependencies. 

On the other hand, the usual way to do things is to match the pypi names, so  ¯\\\_(ツ)_/¯ ‏

Any thoughts @conda-forge/core?

Reply:
For the python runtime, there's no need to have 2 packages.
For `antlr`, we have 2.7.7 in the feedstock. @conda-forge/antlr might be willing to maintain 2.x and 4.x in 2 separate branches. (Only `nco` will be affected at build time. antlr is a build time dependency of `nco`. cc @conda-forge/nco)
@conda-forge/antlr, what do you think?